docs/book/src/jobs.md

```diff
@@ -0,0 +1,42 @@
+# Jobs
```

Comment:
let's put this under developers 

also you need to add the new page to SUMMARY.md so it gets added to the book